Modules/IO/NIFTI/test/itkNiftiReadAnalyzeTest.cxx

```diff
@@ -204,8 +256,67 @@ itkNiftiReadAnalyzeTest(int ac, char * av[])
   {
     return EXIT_FAILURE;
   }
+
+  const auto *                             fPtr = reinterpret_cast<const float *>(LittleEndian_img);
+  itk::ImageRegionConstIterator<ImageType> it(img, img->GetLargestPossibleRegion());
+  it.GoToBegin();
+  for (; !it.IsAtEnd(); ++it, ++fPtr)
+  {
+    //
+    // in the unlikely event we're testing on a big-endian machine, do
+    // byte swapping on floats pulled from the little-endian array.
+    float cur = *fPtr;
+    itk::ByteSwapper<float>::SwapFromSystemToLittleEndian(&cur);
+    if (itk::Math::NotExactlyEquals(it.Get(), cur))
+    {
+      std::cerr << "expected pixel value " << cur << " but found " << it.Get() << std::endl;
+      return EXIT_FAILURE;
+    }
+  }
+  return EXIT_SUCCESS;
+}
+
+int
+itkNiftiAnalyzeCoordinatesTest(char *                                                   av[],
```

Comment:
There is a lot of duplication between this function and `itkNiftiAnalyzeContentsTest`. The old test might even be a complete subset of one of the new tests - perhaps `ITK_COORDINATE_ORIENTATION_RIP`?

Reply:
ok, I removed redundant test